platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -179,6 +191,27 @@ private ImportMigrationConfigurationAdminEntry updateNecessaryConfigurationAdmin
     if (entry.getPid().equals("org.codice.ddf.security.filter.login.Session")) {
       entry.setPid("ddf.security.http.impl.HttpSessionFactory");
     }
+    // this configuration was updated in version 2.13.6
+    // see https://github.com/codice/ddf/pull/4388
+    if (entry.getPid().equals(METACARD_VALIDITY_FILTER_PLUGIN_PID)) {
+      entry = updateMetacardValidityFilterPluginConfiguration(entry);
+    }
+    return entry;
+  }
+
+  private ImportMigrationConfigurationAdminEntry updateMetacardValidityFilterPluginConfiguration(
```

Comment:
❓ Is this logic specific to a given version? Meaning should be applied to all versions of migrations are only those before a certain version. Like does it need to be re-applied if the export was done from the latest version?

Reply:
Ah, yes it is specific to only a few versions! I'll fix that.

Reply:
Technically, this isn't easy. It is specific to a DDF version, but not to a migratable/migration version (since we haven't updated those versions, as configurations have changed). I think we've been avoiding looking at the DDF version in this code because it would entail the migration framework exposing that information to the migratables.

Reply:
Understood. So my question/concern remains. In the absence of being able to rely on the version of the product, are you guaranteed that the above code will work every time and not mess up a perfectly good configuration object that didn't need to be tweaked?

Reply:
Yes, I think it will as long as this comment of mine is addressed 👍 
https://github.com/codice/ddf/pull/5540#discussion_r341216052

Reply:
👍 Thanks for confirming. This rule of thumb has to be understood and confirmed each time. Tests should cover both cases (old versions and new versions) as well.